pkg/tenant/db/users.go

```diff
@@ -37,7 +37,7 @@ func CreateUserResources(ctx context.Context, projectID ulid.ULID, member *Membe
 		ID:       projectID,
 		OrgID:    member.OrgID,
 		TenantID: tenant.ID,
-		Name:     fmt.Sprintf("%s's Project", member.Name),
+		Name:     tenant.Name,
```

Comment:
Should the default project name be the same as the tenant name? Or something else?

Reply:
Same as the tenant name/org name works for me.